### PR TITLE
Update docs for deferred auth

### PR DESCRIPTION
**Pull request by ekoleda-codaio**
- Update the text and screen recording in the Authentication.
- Update the text and images in the OAuth2 tutorial.
- Create an `optimize-video` build rule to capture `ffmpeg` configurations needed to optimize videos for embedded animations.
- Optimize some two-way sync images that were missed in previous PRs.

### CODE REVIEW

File: docs/guides/basics/authentication/index.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Packs that support per-user authentication require some additional setup. Users
 
 ### Connecting an account
 
-The listing for the Pack will display a **Sign in to install** button in place of the usual **Install** button. Upon clicking it the user will immediately be launched into a sign-in flow, which varies depending on the type of authentication used and the service the user is connecting to. After the sign-in flow is complete Coda will ask the user if that connected account should be shared with other users in the same doc.
+The first time a user adds a building block that requires authentication they will be shown a **Connect an account** button. Upon clicking it the user will be launched into a sign-in flow, which varies depending on the type of authentication used and the service the user is connecting to.
```
Comment (patrick-codaio) [inline docs/guides/basics/authentication/index.md:22]: teeeechnically they might not get a button - a table with getSchema will auto-prompt auth before performing the insertion - but i don't think that's something you need to clarify here.

File: docs/images/two_way_autocomplete.png
Comment (patrick-codaio) [inline docs/images/two_way_autocomplete.png:1]: pretty much identical... not sure if you planned more of a change here?

File: docs/guides/basics/authentication/index.md (hunk 1 above)
Comment (ekoleda-codaio) [reply docs/guides/basics/authentication/index.md:22]: Good to know! But I agree we probably don't need that level of detail here.

File: docs/images/two_way_autocomplete.png
Comment (ekoleda-codaio) [reply docs/images/two_way_autocomplete.png:1]: This is just a file size delta. Running the image optimizer is a manual process, and I forgot to do so when I added this image originally.

File: docs/images/two_way_enable.png
Comment (patrick-codaio) [inline docs/images/two_way_enable.png:1]: This screenshot is out of date - "edit mode" is now "write mode" for instance

Comment (ekoleda-codaio) [reply docs/images/two_way_enable.png:1]: Ya 😔 I've been noticing that a lot of my screenshots and animations are out of date (one still has the "Explore" menu!) I'm thinking about doing an end-of-year pass over the whole documentation, but I think it probably makes sense to spruce up the two-way sync page sooner, given the likely interest.

If you don't mind, I'll tackle that in a follow-up PR.

Comment (patrick-codaio) [reply docs/images/two_way_enable.png:1]: Ah, ok

**Review by patrick-codaio — APPROVED** [review]

**Pull request merged**